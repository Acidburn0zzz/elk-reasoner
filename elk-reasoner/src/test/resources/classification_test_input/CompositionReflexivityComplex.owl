Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Prefix( rdfs: = <http://www.w3.org/2000/01/rdf-schema#> )
Ontology(<www.example.org#composition_reflexivity_complex>
Annotation(rdfs:label "Due to reflexivity of :R, ontology implies SubObjectPropertyOf(ObjectPropertyChain(:S :T) :H) and hence SubClassOf(:A :B)")

ReflexiveObjectProperty(:R)
SubObjectPropertyOf(ObjectPropertyChain(:R :S :T) :H)
SubObjectPropertyOf(ObjectPropertyChain(:H :V) :W)

SubClassOf(:A ObjectSomeValuesFrom(:S :B))
SubClassOf(:B ObjectSomeValuesFrom(:T :C))
SubClassOf(:C ObjectSomeValuesFrom(:V :D))

EquivalentClasses(ObjectSomeValuesFrom(:W :D) :E)

)