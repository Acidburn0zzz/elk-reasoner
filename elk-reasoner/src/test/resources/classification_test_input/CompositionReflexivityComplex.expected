Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#composition_reflexivity_complex#taxonomy>

SubClassOf(:A :E)
Declaration(Class(:B))
Declaration(Class(:C))
Declaration(Class(:D))

)